Gems/Camera/Code/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ ly_add_target(
 ly_add_target(
     NAME Camera ${PAL_TRAIT_MONOLITHIC_DRIVEN_MODULE_TYPE}
     NAMESPACE Gem
+    OUTPUT_NAME Camera.Gem
```

Comment:
Is this necessary? this changes the name of the output file for this target
Same for the case below

Reply:
I can revert this, I think I did this as I looked at how some other tests load the PhysX library and it had this specified, I'll just use the existing name instead

Reply:
Now updated 👍 

One thing I did spot though is even though in `EditorLib.Camera.Tests`, `Gem::Camera.Editor` is marked as a private build dependency, if I delete the .dll of `Camera.Editor.dll` and then build `EditorLib.Camera.Tests`, it does not appear as though the Camera Gem is also built. Is this expected or am I missing something? Thanks!